Microsoft.Azure.CosmosRepository/src/InMemoryRepository.cs

```diff
@@ -14,13 +14,18 @@
 using Microsoft.Azure.CosmosRepository.Builders;
 using Microsoft.Azure.CosmosRepository.Extensions;
 using Microsoft.Azure.CosmosRepository.Internals;
+using Newtonsoft.Json;
 
 namespace Microsoft.Azure.CosmosRepository
 {
     /// <inheritdoc/>
     internal class InMemoryRepository<TItem> : IRepository<TItem> where TItem : class, IItem
     {
-        internal ConcurrentDictionary<string, TItem> Items { get; } = new();
+        internal ConcurrentDictionary<string, string> Items { get; } = new();
+
+        internal string SerialiseItem(TItem item) => JsonConvert.SerializeObject(item);
```

Comment:
For consistency with the delegated API that is being called, could we please rename all of these to `Serialize` and `Deserialize`? 😄 

```suggestion
        internal string SerializeItem(TItem item) => JsonConvert.SerializeObject(item);
```